srcpkgs/psfiles/template

```diff
@@ -0,0 +1,11 @@
+pkgname=psfiles
+version=0.1.0
+revision=1
+archs="x86_64"
```

Comment:
remove archs restriction even if you can't test, let CI build it.

Reply:
Upstream support only x86_64. Okay, enable musl by changing to x86_64* and leave comment.

Reply:
@Chocimier fixed